drivers/volume/portworx/portworx.go

```diff
@@ -1503,10 +1503,10 @@ func (d *portworx) ValidateCreateVolume(volumeName string, params map[string]str
 				return errFailedToInspectVolume(volumeName, k, requestedSpec.AggregationLevel, vol.Spec.AggregationLevel)
 			}
 			/* Ignore shared setting.
-		case api.SpecShared:
-			if requestedSpec.Shared != vol.Spec.Shared {
-				return errFailedToInspectVolume(volumeName, k, requestedSpec.Shared, vol.Spec.Shared)
-			}
+			case api.SpecShared:
```

Comment:
Is this indentation correct ?

Reply:
yes, this is correct , its aligned right as the code was commented